.github/workflows/call-build-windows.yaml

```diff
@@ -112,6 +112,27 @@ jobs:
         with:
           arch: ${{ matrix.config.arch }}
 
+      - name: Get gzip command w/ chocolatey
+        uses: crazy-max/ghaction-chocolatey@v2
+        with:
+          args: install gzip -y
```

Comment:
What's the reason for needing gzip?

Reply:
The reason is `actions/cache/save@v3` is heavily depends on gzip command actually...